components/tyk-bootstrap/templates/bootstrap-post-install.yaml

```diff
@@ -26,7 +26,9 @@ spec:
         {{ toYaml .Values.bootstrap.podAnnotations.postInstallPodAnnotations | nindent 8 }}
         {{- end }}
     spec:
-      serviceAccountName: k8s-bootstrap-role
+    {{- if and .Values.bootstrap .Values.global.serviceAccountName }}
```

Comment:
do we need to check for `.Values.bootstrap`? i do not think that we have such field. can you please double check again? Also, in my opinion, we do not need if condition here, we can populate `spec.serviceAccountName` of this Job directly according to `serviceAccountName` field.

Reply:
before the SA is created for the job, there's a check for the boostrap configuration. This is just taking precautions to ensure that the sa exists before it is attached to it.